### ISSUE TITLE
RTL on Raspiblitz: Cannot unlock wallet?

### ISSUE THREAD

**Issue by lets-fuckin-do-it** [Ride-The-Lightning/RTL#550]
New to LN and Raspiblitz but managed to build, install one send funds, open channels using Raspiblitz LN command line tool.
Now that I am confortable I wanted to have a look at RTL and its legendary beautiful GUI. 
Although I can access RTL (http://192.168.68.102:3000/rtl) and login with my password (what they call 'password B' in Raspiblitz) I am immediatly asked to Unlock Wallet (what they call 'password C' in Raspiblitz) which returns an error message pop-up (screenshot attached).
Note that my node works, I have channels running, all synchronized, all looks good apart from I cannot start RTL (unlock wallet) for some reason.
Any help welcome

Raspiblitz 1.6.1
RTL 0.9.3-beta
lnd 0.11.1-beta commit=v0.11.1-beta, build=production

![2020-11-30_00-34-42](https://user-images.githubusercontent.com/26678816/100559841-3afb9380-32b4-11eb-90f0-61a65fbe45f0.jpg)

**Comment by saubyk** [comment]
Hi, it appears that RTL is not able to communicate with LND.
Its like a config issue, would you be able to share your RTL-Config.json file? Do remember to mask any sensitive information you may have on it.

Thanks

**Comment by lets-fuckin-do-it** [comment]
Thanks for getting back to me. I don't see a RTL-Config.json file in /user/admin
```
$ pwd
/home/admin
$ find . -name "RTL-Config.json"
```
-> returns nothing. 

Accessing RTL http://myraspiblitz:3000/rtl/settings (>LND config) gives me:

```
[Application Options]
debuglevel=debug
maxpendingchannels=5
alias=latribudunord
color=
nat=false
rpclisten=0.0.0.0:10009
restlisten=0.0.0.0:8080
[Bitcoin]
bitcoin.active=1
bitcoin.node=bitcoind
bitcoin.mainnet=1
[Bitcoind]
bitcoind.rpcuser=raspibolt
bitcoind.rpcpass=*************
bitcoind.rpchost=127.0.0.1
bitcoind.zmqpubrawblock=tcp://*:28332
bitcoind.zmqpubrawtx=tcp://*:28333
[autopilot]
autopilot.active=1
autopilot.maxchannels=5
autopilot.allocation=0.6
[Tor]
tor.password=formyeyesonly

```

**Comment by lets-fuckin-do-it** [comment]
Where can the config file be located?

**Comment by saubyk** [comment]
Hi @openoms @rootzoll
Would you be able to advise where the RTL config settings would be located on Raspiblitz?

In this case RTL is not able to connect with LND and we need to examine the config settings to troubleshoot.
Thanks

**Comment by lets-fuckin-do-it** [comment]
Found it: It is under /home/rtl/RTL in RaspiBlitz
I read a bit more and listened to RaspiBlitz technical update (it is great: https://www.youtube.com/watch?v=ScFTGLC2ZlY&list=WL&index=433) - not that it helped resolve the issue but made me realise:
The issue was: Because I played around with wallets (created/deleted) my RTL config file was pointing to a wallet that did not exist anymore. 
Would be nice if RTL could pop-up a message "the wallet blabla' (from the RTL config file) does not exist instead of "Error Code 500" message (my previous screenshot).
Point closed. Thank you very much Saubyk, have a great day.
